cmd/kubeadm/app/util/runtime/runtimefakes/fake_impl.go

```diff
@@ -0,0 +1,720 @@
+/*
```

Comment:
if we agree to not use the generator, this fake implementation can just reside in runtime_test.go and be 10% of the generated complexity.

Reply:
Sounds good, I switched to a manual fake implementation.